src/EventListener/MergeHttpHeadersListener.php

```diff
@@ -11,13 +11,16 @@
 namespace Contao\CoreBundle\EventListener;
 
 use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\CoreBundle\HttpKernel\Header\HeaderStorageInterface;
+use Contao\CoreBundle\HttpKernel\Header\NativeHeaderStorage;
 use Symfony\Component\HttpFoundation\Response;
```

Comment:
`HttpFoundation` comes before `HttpKernel`. 😄

Reply:
Yeah, but Symfony does not come before Contao 😉

Reply:
LOL